internal/lib/testutils/minirunner/minirunner.go

```diff
@@ -24,7 +24,7 @@ type MiniRunner struct {
 	Fn              func(ctx context.Context, state *lib.State, out chan<- metrics.SampleContainer) error
 	SetupFn         func(ctx context.Context, out chan<- metrics.SampleContainer) ([]byte, error)
 	TeardownFn      func(ctx context.Context, out chan<- metrics.SampleContainer) error
-	HandleSummaryFn func(context.Context, *lib.Summary) (map[string]io.Reader, error)
+	HandleSummaryFn func(context.Context, *lib.LegacySummary) (map[string]io.Reader, error)
```

Comment:
Shouldn't we also use the new signature here? :thinking:

Reply:
I think the reason why I didn't do it is because I think it is used nowhere, but yeah, it's a good idea overall.

Reply:
Done @ https://github.com/grafana/k6/pull/4089/commits/58e3d021ea7509bfa7aba35b0babefb7c54e04cc